src/services/api/bookService.ts

```diff
@@ -0,0 +1,11 @@
+import axios from '../../axios/axiosConfig';
+
+const getBook = (id: string | undefined) => {
+  return axios.get(`/book/${id}`).then((res) => {
```

Comment:
What are you doing here if id is undefined, for me it is better not to use undefined